contracts/axelar-gas-service/src/test.rs

```diff
@@ -22,14 +26,87 @@ fn setup_env<'a>() -> (Env, Address, Address, AxelarGasServiceClient<'a>) {
 
     let client = AxelarGasServiceClient::new(&env, &contract_id);
     let gas_collector: Address = Address::generate(&env);
-    client.initialize(&gas_collector);
```

Comment:
Not sure why you move `client.initialize` outside of setup_env(). Does it have to be moved out of the function?

Reply:
good point, moved it back. I think I was planning to test `NotInitialized`  (so needed to setup without initializing), but didn't get around to it. Even then it's worth just duplicating the preamble for the one test instead